defender/datadog_forta_detection_bot_health/autotask-1/index.js

```diff
@@ -52,8 +52,17 @@ function parseAgentInformationResponse(response) {
 }
 
 function parseMetricsResponse(response, currentTimestamp) {
-  // const { data: { data: { getAgentMetrics: { metrics } } } } = response;
-  const { data: { data: { getAgentMetrics: { chains: [{ metrics }] } } } } = response;
+  const chains = response?.data?.data?.getAgentMetrics?.chains;
+  if (chains === undefined || chains.length === 0) {
+    return {};
+  }
+  let metrics = [];
```

Comment:
I think this could be simplified like this:
`const metrics = chains.find(chain => chain.chain_id === 1)?.metrics || [];`